### PR TITLE
fix(cp): change selectable to pointer

### PR DESCRIPTION
**Pull request by bugaolengdeyuxiaoer**
#### Which issue(s) this PR fixes:
Fixes #

#### Specified Reivewers:
/assign @your-reviewer

### CODE REVIEW

File: providers/component-protocol/components/list/model.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ type (
 		Description      string      `json:"description,omitempty"`
 		BackgroundImgURL string      `json:"backgroundImgURL,omitempty"`
 		KvInfos          []KvInfo    `json:"kvInfos,omitempty"`
-		Selectable       bool        `json:"selectable,omitempty"`
+		Selectable       *bool       `json:"selectable,omitempty"`
```
Comment (sfwn) [inline providers/component-protocol/components/list/model.go:62]: why *bool, please tell me your scenarios.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: providers/component-protocol/components/list/model.go (hunk 1 above)
Comment (bugaolengdeyuxiaoer) [reply providers/component-protocol/components/list/model.go:62]: json can not identify false and no value of bool field. for fronted , selectable is required.

Comment (sfwn) [reply providers/component-protocol/components/list/model.go:62]: Yes, I want to know the different between `false` and `no value`.

Comment (bugaolengdeyuxiaoer) [reply providers/component-protocol/components/list/model.go:62]: I mean for fronted, true is default value and false is required if item can not be selected, but if `selectable bool` nothing can be transfer .

**Review by sfwn — APPROVED** [review]

**Pull request merged**